cfme/v2v/migration_plans.py

```diff
@@ -442,6 +440,18 @@ def get_plan_vm_list(self):
         request_details_list = view.migration_request_details_list
         return request_details_list
 
+    METHODS_PER_STATE = {'Started': plan_started,
+                         'In_Progress': in_progress,
+                         'Completed': completed,
+                         'Successful': successful}
+
+    def wait_for_state(self, state):
+        if state in MIGRATION_STATES:
+            return bool(self.METHODS_PER_STATE[state](self))
```

Comment:
I guess you need to handle exceptions here since your wait_for methods will raise exceptions.